docs/decisions/0009-persistence-and-retries-for-events.rst

```diff
@@ -0,0 +1,25 @@
+Persistence and retries for events
+==================================
+
+Status
+------
+
+Pending
+
+Context
+-------
+
+`event-routing-backends` transmits events to configured Learning Record Consumer (LRC) via http protocol in near real-time. A strategy needs to be implemented to handle the case when an LRC's link is down.
```

Comment:
[Nit] Learning Record Consumer is an xAPI term. Is it being used correctly in this context?  My understanding is that an LRC is a client of a Learning Record Store. Not a client of our event emission.

Reply:
Agreed and changed.